lms/static/js/spec/instructor_dashboard/student_admin_spec.js

```diff
@@ -129,7 +129,58 @@ define(['jquery', 'coffee/src/instructor_dashboard/student_admin', 'js/common_he
                 expect(studentadmin.$request_response_error_ee.text()).toEqual(full_error_message);
             });
 
-            it('binds delete student state for entrance exam ajax call and the result will be success', function() {
+            it('initiates skip entrance exam when button is clicked', function() {
+                studentadmin.$btn_skip_entrance_exam.click();
+                // expect error to be shown since student identifier is not set
+                expect(studentadmin.$request_response_error_ee.text()).toEqual(gettext("Please enter a student email address or username."));
+
+                var success_message = gettext("Student '{student_id}' is marked to skip entrance exam.");
+                var full_success_message = interpolate_text(success_message, {
```

Comment:
Suggested revision: 

This student ('{student_id}') will skip the entrance exam.